### DISCUSSION TITLE
Uninvited users adding pages to PubPub community

### DISCUSSION THREAD

**Discussion by nelscity** [knowledgefutures/pubpub#1498] [Support]
I was looking at the analytics for [my pubpub](https://urban.pubpub.org/) and there are several users who have added themselves as users somehow, even though they do not show up in my members list. How does this happen? Are you aware that this can be done?

[Example 1](https://urban.pubpub.org/user/hithyshi-hithu)
[Example 2](https://urban.pubpub.org/user/john-solomon-2)
[Example 3](https://urban.pubpub.org/user/hope-santini)

**Comment by gabestein** [comment]
Hi Nels — this is a known issue related to how we build author pages. Rest assured that these users do not have permissions in your community, their user pages just show up with your community's branding. However, we know this can cause SEO and trust issues, and we are planning to fix it in the current sprint as described here: https://github.com/pubpub/pubpub/issues/1429
